config/lib/mongoose.js

```diff
@@ -100,19 +138,54 @@ module.exports.disconnect = function (callback) {
 
 module.exports.dropDatabase = function (connection, callback) {
   if (process.env.NODE_ENV === 'production') {
-    console.error('You cannot drop database in production mode!');
+    log('error', 'You cannot drop database in production mode!');
     return process.exit(1);
   }
 
   connection.dropDatabase(function (err) {
     if (err) {
-      console.error('Failed to drop database', err);
+      log('error', 'Failed to drop database', {
+        error: err
+      });
     } else {
-      console.log('Successfully dropped database:', connection.db.databaseName);
+      log('info', 'Successfully dropped database: ' + connection.db.databaseName);
     }
 
     if (callback) {
       callback(err);
     }
   });
 };
+
+module.exports.ensureIndexes = function (modelNames) {
+  return new Promise(function (resolve, reject) {
+    // assuming openFiles is an array of file names
+    async.each(modelNames, function (modelName, callback) {
+      mongoose.connection.model(modelName).ensureIndexes(function (error) {
+        if (error) {
+          log('error', 'Indexing Mongoose Schema failed', {
+            model: modelName,
+            error: error
+          });
+          callback(error);
+        } else {
+          log('info', 'Indexed Mongoose Schema ' + modelName);
+          callback();
+        }
+      });
+    }, function (error) {
+      // if any of the file processing produced an error
+      if (true || error) {
```

Comment:
`true` leftover from debugging? this will always complain about error.

Reply:
Yep! Totally. Dealt with in https://github.com/Trustroots/trustroots/pull/1035/commits/14c73511da35cb2b9c91f5c1a18f176bc1b4a782

Reply:
:) ok. Seems I don't have the most recent version locally.

Reply:
I mean, I dealt it with after you pointed it out. ;-) You spot it well, I cleaned it out fast. ;-)